peekingduck/pipeline/nodes/input/live.py

```diff
@@ -24,7 +24,47 @@
 
 
 class Node(AbstractNode):
-    """Node to receive livestream as inputs."""
+    """Node to receive livestream as inputs.
+
+    Inputs:
+        None
+
+    Outputs:
+        |img|
+
+        |filename|
+
+        |pipeline_end|
+
+        |saved_video_fps|
+
+    Configs:
+        fps_saved_output_video (:obj:`int`): **default = 10**
+
+            FPS of the mp4 file after livestream is processed and exported.
+            FPS dependant on running machine performance.
```

Comment:
should be `dependent` instead of `dependant`

Reply:
Thanks. I have made the changes